.github/workflows/build.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
           java-version: 18
       - uses: gradle/gradle-build-action@v2
 
-      - run: ./gradlew build dokkaHtmlMultiModule verifyPaparazziDebug --parallel
+      - run: ./gradlew assemble lint allTests verifyPaparazziDebug dokkaHtmlMultiModule --parallel
```

Comment:
`build` doesn't run the `UIViewChildren` tests, but `allTests` does. Replaced `build` with separate `assemble` and `lint` since `build` also runs the tests which isn't necessary with `allTests`.

Reply:
We need to keep build or at least run check (but build is just assemble+check so might as well keep build). With this change you've removed spotless from the CI codepath, for example.

Seems like that's a severe bug if so and needs filed on Kotlin. check task needs to run all tests!

Reply:
Gotcha, I'll revert back to `build` and submit an issue to the tracker.